src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -1135,7 +1139,7 @@ function getStackingConfig(stackByAttribute: any, options: IChartConfig): string
      * 3) If chart is column or bar chart and 'Stack Measures' is enabled
      */
     const isStackByChart = stackByAttribute && supportsStacking;
-    const isAreaChartWithEnabledStacking = isAreaChart(type) && enableAreaChartStacking(stacking);
+    const isAreaChartWithEnabledStacking = isAreaChart(type) && enableAreaChartStacking(stackMeasures, stacking);
```

Comment:
consider to move `isAreaChart` into `enableAreaChartStacking` function...
also `enableAreaChartStacking` could consume whole `options` :)